services/core-web/src/components/mine/Permit/MinePermitTable.tsx

```diff
@@ -96,94 +89,6 @@ const finalApplicationPackage = (amendment) => {
   return finalAppPackageCore.concat(finalAppPackageImported);
 };
 
-const renderDeleteButtonForPermitAmendments = (record) => {
```

Comment:
Very happy with these changes.  I really dislike the previous pattern used here.